containers/Makefile.am

```diff
@@ -55,3 +55,17 @@ ws-container:
 ws-container-shell:
 	sudo docker run -ti --rm --publish=9001:9090 \
 		cockpit/ws /bin/bash
+
+UNIT_TESTS_DIR = $(srcdir)/containers/unit-tests
+
+unit-tests-container:
+	sudo docker build -t cockpit/unit-tests $(UNIT_TESTS_DIR)
+	sudo docker tag  cockpit/unit-tests:latest docker.io/cockpit/unit-tests:latest
+	sudo docker build --build-arg arch=i386 -t cockpit/unit-tests:i386 containers/unit-tests/
+	sudo docker tag  cockpit/unit-tests:i386 docker.io/cockpit/unit-tests:i386
+
+unit-tests-container-run:
+	sudo docker run -t --rm -v $(abs_srcdir):/source:ro cockpit/unit-tests
```

Comment:
I think it makes sense to add `-i` here as well. Terminating the test run with `Ctrl-C` only detaches the container (but keeps it running) when it is not given. I think that's quite surprising when running a command from a Makefile.

Reply:
Thanks for the hint, that works nicely with ^C! Done, and changed in the README too.

Force-pushed, and I also re-enabled the i386 run *if* there is a `containers/unit-tests`, i. e. only for this PR (and every PR once this lands).